extension/src/ALObject/ALElementTypes.ts

```diff
@@ -620,7 +637,7 @@ export class ALObject extends ALControl {
     return !deleteAllowed && !insertAllowed && !modifyAllowed;
   }
   public get publicAccess(): boolean {
-    const val = this.getProperty(ALPropertyType.access, "public");
+    const val = this.getProperty(ALPropertyType.access, "public") as string;
     return val.toLowerCase() === "public";
```

Comment:
Not that it really brings any value but since I saw it; this could be one line.

Reply:
I think it helps debugging when it's on two lines...